sbt-plugin/src/main/scala/scala/scalajs/sbtplugin/cross/CrossType.scala

```diff
@@ -40,25 +40,23 @@ object CrossType {
 
   object Full extends CrossType {
     def projectDir(crossBase: File, projectType: String): File =
-      crossBase.getAbsoluteFile / projectType
+      crossBase / projectType
```

Comment:
Well, the project directory should be absolute at this point. We are referring to one physical directory on the filesystem. It seems really strange that removing these fixes anything.

I'll need to take a closer look at this tonight or tomorrow. Please do not merge this unless you know exactly what is going on here.